nix/overte.nix

```
@@ -0,0 +1,176 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  pkg-config,
+  autoPatchelfHook,
+
+  glad,
+  gif_creator,
+  polyvox,
+  artery-font-format,
+  cgltf,
+  etc2comp,
+  openssl,
+  libGL,
+  glm,
+  nlohmann_json,
+  tbb_2022,
+  nodejs,
+  webrtc-audio-processing,
+  nvidia-texture-tools,
+  openexr,
+  draco,
+  bullet,
+  discord-rpc,
+  openvr,
+  openxr-loader,
+  SDL2,
+  libopus,
+  libsForQt5,
+  libv8,
+
+  scribe,
+  glslang,
+  spirv-tools,
+  spirv-cross,
+  python3,
+
+  buildClient ? true,
+  buildServer ? true,
+  buildTools ? false,
+}:
```

```suggestion
{
  # build tools
  lib,
  stdenv,
  cmake,
  pkg-config,
  autoPatchelfHook,

  # dependencies
  glad,
  gif_creator,
  polyvox,
  artery-font-format,
  cgltf,
  etc2comp,
  openssl,
  libGL,
  glm,
  nlohmann_json,
  tbb_2022,
  nodejs,
  webrtc-audio-processing,
  nvidia-texture-tools,
  openexr,
  draco,
  bullet,
  discord-rpc,
  openvr,
  openxr-loader,
  SDL2,
  libopus,
  libsForQt5,
  libv8,

  # tools for shader compilation
  scribe,
  glslang,
  spirv-tools,
  spirv-cross,
  python3,

  # build options
  buildClient ? true,
  buildServer ? true,
  buildTools ? false,
}:
```

Could you add comments here? It's hard to tell what this does, especially considering it is in unnamed curly braces.